pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -57,9 +61,8 @@ def pygrb_initialize_plot_parser(description=None, version=None):
     formatter_class = argparse.ArgumentDefaultsHelpFormatter
     parser = argparse.ArgumentParser(description=description,
                                      formatter_class=formatter_class)
+    add_common_pycbc_options(parser)
```

Comment:
Outwith this change, all the changes in `pycbc` directory are:
 - adding the named logger to the module
 - updating `logging` --> `logger` to use the named logger
 - some reordering/reformatting of imports